data/blog/mobile-app-launch-profiling.md

```diff
@@ -0,0 +1,46 @@
+---
+title: 'Mobile App Launch Profiling'
+date: '2024-04-17'
+tags: ['mobile', 'ios', 'android', 'profiling']
+draft: false
+summary: "See what's happening in your app before your first line of code can even run."
+images: ['/images/mobile-app-launch-profiling/launch-profile.png']
+layout: PostLayout
+canonicalUrl: mobile-app-launch-profiling
+authors: ['andrewmcknight']
+---
+
+If you use Sentry’s Performance monitoring on Cocoa and Android, you also have the ability to see [profiles](https://docs.sentry.io/product/profiling/) of how your app runs in the wild on real end user devices. Any time an automatic or manual transaction is recorded, that profiling data is attached. But are there times where you can’t record a transaction? 🤔
+
+# App launch performance
+
+<aside>
+💡 This post will focus on our Cocoa app launch profiling feature. You can read more about the Android counterpart by heading over to that area of our docs: https://docs.sentry.io/platforms/android/profiling/#app-start-profiling
```

Comment:
React Native has its own profiler, right? I won't leverage this capability? iOS apps built with Flutter and .NET use the cocoa SDK for profiling as-is:

```suggestion
💡 This post will focus on our iOS app launch profiling feature. Including iOS apps built on [Flutter](https://docs.sentry.io/platforms/flutter/profiling/) and [.NET](https://docs.sentry.io/platforms/dotnet/guides/maui/profiling/). You can read more about the Android counterpart by heading over to [that area of our docs](https://docs.sentry.io/platforms/android/profiling/#app-start-profiling).
```

Reply:
Hm I'm just not sure, I haven't tested either of those.

Reply:
They are both on the docs and we're receiving a lot of data (from Flutter in particular, not so much on .NET).

Reply:
Yeah sounds good.